src/jit/codegencommon.cpp

```diff
@@ -10241,6 +10241,62 @@ void CodeGen::genCaptureFuncletPrologEpilogInfo()
 
 // Look in CodeGenArm64.cpp
 
+#elif defined(_TARGET_X86_)
+
+/*****************************************************************************
+ *
+ *  Generates code for an EH funclet prolog.
+ */
+
```

Comment:
936ebb7 adds TODO (I'm not sure about (1) and (3), and thus it adds only (2) as TODO).